src/js/models/figure_model.js

```diff
@@ -132,6 +132,16 @@
                     }
                 });
             }
+            if (v < 3) {
+                console.log("Transforming to VERSION 3");
```

Comment:
to be removed

Reply:
@jburel I've added these console.logs for all the upgrade steps since I think they might help debugging at some point. Don't think this limited logging does any harm.